lars-reimann/api-editor
# feat: Automatisierte Herleitung von Boundary Annotations

Closes #445.

### Summary of Changes



### Screenshots (if necessary)



### Testing instructions

1. Open a terminal in `package-parser`
2. Run `parse-package generate -a .\tests\data\api_data.json -u .\tests\data\usage_data.json -o .\out\out.json`
3. Check the boundary annotations that are created in `out\out.json`.
